### PR TITLE
README update

### PR DESCRIPTION
**Pull request by matejoravec**

### CODE REVIEW

**Comment by guderkar** [comment]
I think there should also be how to set entity in configuration yaml and also cell with widgets factory is missing

File: README.md [hunk 1]
```diff
@@ -12,55 +12,101 @@ poetry add feature-store-bundle
 
 # Getting started
 
+1. Define entities in `config.yaml` of your Daipe project
```
Comment (ds-lukaslangr) [inline README.md:15]: I would probably just delete the README contents completely, there is no way for us to keep it up to date at all times

Comment (ds-lukaslangr) [reply README.md:15]: and nobody ever reads it :)

**Review by ds-lukaslangr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (matejoravec) [reply README.md:15]: How about you @guderkar ? :)

**Pull request merged**